File: samples/snippets/csharp/VS_Snippets_Remoting/RemotingConfiguration_IsRemotelyActivatedClientType2_Client/CS/RemotingConfiguration_IsRemotelyActivatedClientType2_Server.cs
using System;
using System.Runtime.Remoting;
using System.Runtime.Remoting.Channels;
using System.Runtime.Remoting.Channels.Tcp;

public class ServerClass
{
   static void Main()
   {
      ChannelServices.RegisterChannel(new TcpChannel(8085), false);
      RemotingConfiguration.RegisterActivatedServiceType(typeof(MyServerImpl));
      Console.WriteLine("Press enter to stop this process.");
      Console.ReadLine();
   }
}

File: samples/snippets/cpp/VS_Snippets_Remoting/RemotingConfiguration_IsRemotelyActivatedClientType2_Client/CPP/remotingconfiguration_isremotelyactivatedclienttype2_server.cpp


#using <System.Runtime.Remoting.dll>
#using <System.dll>
#using <RemotingConfiguration_IsRemotelyActivatedClientType2_Shared.dll>

using namespace System;
using namespace System::Runtime::Remoting;
using namespace System::Runtime::Remoting::Channels;
using namespace System::Runtime::Remoting::Channels::Tcp;

int main()
{
   ChannelServices::RegisterChannel( gcnew TcpChannel( 8085 ), false );
   RemotingConfiguration::RegisterActivatedServiceType( MyServerImpl::typeid );
   Console::WriteLine( "Press enter to stop this process." );
   Console::ReadLine();
}
